genrl/bandit/agents/cb_agents/common/neural.py

```diff
@@ -0,0 +1,104 @@
+from typing import List, Optional, Tuple
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from genrl.bandit.agents.cb_agents.common.transition import TransitionDB
+
+
+class NeuralBanditModel(nn.Module):
+    """Neural Network used in Deep Contextual Bandit Models.
+
+    Args:
+        context_dim (int): Length of context vector.
+        hidden_dims (List[int], optional): Dimensions of hidden layers of network.
+        n_actions (int): Number of actions that can be selected. Taken as length
+            of output vector for network to predict.
+        init_lr (float, optional): Initial learning rate.
+        max_grad_norm (float, optional): Maximum norm of gradients for gradient clipping.
+        lr_decay (float, optional): Decay rate for learning rate.
+        lr_reset (bool, optional): Whether to reset learning rate ever train interval.
+            Defaults to False.
+        dropout_p (Optional[float], optional): Probability for dropout. Defaults to None
+            which implies dropout is not to be used.
+
+    Attributes:
+        use_dropout (bool): Indicated whether or not dropout should be used in forward pass.
+    """
+
+    def __init__(self, **kwargs):
+        super(NeuralBanditModel, self).__init__()
+        self.context_dim = kwargs.get("context_dim")
+        self.hidden_dims = kwargs.get("hidden_dims")
+        self.n_actions = kwargs.get("n_actions")
```

Comment:
Don't use kwargs here, you can use the actual variable names and and use kwargs only if needed

Reply:
Then I get code climate issue for too many args